### DISCUSSION TITLE
What does this parameter mean "maxFeedCapacity"?

### DISCUSSION THREAD

**Discussion by VlaK0r** [Athou/commafeed#1118] [Q&A]
Hello!
What does this parameter mean?
The number of unread feeds or the total number of all feeds?

**Comment by Athou** [comment]
It's a database cleanup value used to keep the database size in check.

It means that if a feed is refreshed and entries are added to the database, old entries will be deleted from the database until the number of entries for that feed is lower than this value.

**Reply by VlaK0r** [reply]
And if the record is tagged, then it will still be deleted?

**Reply by VlaK0r** [reply]
If an entry is tagged, will it also be deleted?
